README.md

```diff
@@ -14,13 +14,13 @@ This book contains some common **DOs** & **DON'Ts** for Open Source software.
 
 The Open Source community is thriving. Each day the number of Open Source projects grows, as does the army of contributors that maintain them. While this is exciting for the industry, it can be daunting as a developer new to the community. This book aims to provide some tips for newcomers to help them avoid the pitfalls of Open Source development and learn from the community's collective wisdom.
 
-As the ancient proverb goes, _Time and tide [and technology] wait for no man_. And to the best of our ability, neither will this book. Remember to check the version number for updates!
+As the ancient proverb goes, Time and tide [and technology] wait for no man. And to the best of our ability, neither will this book. Remember to check the version number for updates!
 
 We would love your help in keeping this book updated. Your comments, suggestions and pull requests are most welcome.
 
-Open Source is dominating the software industry. Its champions include well known organisations like Facebook, Twitter and Netflix, but more significantly, an army of passionate individual developers around the world. Their efforts have impacted almost every part of computer science, culminating in millions of open source projects, with billions of lines of code!
+Open Source is dominating the software industry. Its champions include well known organizations like Facebook, Twitter and Netflix, but more significantly, an army of passionate individual developers around the world. Their efforts have impacted almost every part of computer science, culminating in millions of Open Source projects, with billions of lines of code!
```

Comment:
The spelling of `organisation` is correct for English, its American that uses a `z` instead of an `s`.

You are correct, `Open Source` is not consistent, thank you 👍

Reply:
Oh, thank you for clarifying.